algorithms/src/msm/variable_base/cuda.rs

```diff
@@ -224,10 +224,9 @@ pub(super) fn msm_cuda<G: AffineCurve>(
         unimplemented!("trying to use cuda for unsupported curve");
     }
 
-    if bases.len() < scalars.len() {
-        scalars = &scalars[..bases.len()];
-    } else if bases.len() > scalars.len() {
-        bases = &bases[..scalars.len()];
```

Comment:
@raychu86 the equal case was not previously considered. Can you sanity check this?

Reply:
The equals case was not considered before because there was no need to truncate. 

The new changes will effectively have the same output. Looks fine to me.